scripts/bootstrapper/run.ps1

```diff
@@ -164,14 +169,8 @@ if (!$Channel) { $Channel = 'dev' }
 if (!$ToolsSource) { $ToolsSource = 'https://aspnetcore.blob.core.windows.net/buildtools' }
 
 # Execute
-
 $korebuildPath = Get-KoreBuild
-Import-Module -Force -Scope Local (Join-Path $korebuildPath 'KoreBuild.psd1')
 
-try {
-    Install-Tools $ToolsSource $DotNetHome
-    Invoke-RepositoryBuild $Path @MSBuildArgs
-}
-finally {
-    Remove-Module 'KoreBuild' -ErrorAction Ignore
-}
+$koreBuildproj = Join-Paths $korebuildPath ("tools", "KoreBuild.Console.dll")
+
+& dotnet $koreBuildproj $Command --toolsSource=$ToolsSource --dotNetHome=$DotNetHome --repoPath=$Path $Arguments
```

Comment:
As above, this isn't going to work. We don't have a gurantee that dotnet is available, or that the right shared runtime is installed. This needs to happen first.